config-templates/azure/azuredisk-csi-driver/1.23.0/deployment.yaml

```diff
@@ -0,0 +1,667 @@
+      - apiGroups: [""]
+        resources: ["pods"]
+        verbs: ["get", "list", "watch"]
+  - kind: ClusterRoleBinding
+    apiVersion: rbac.authorization.k8s.io/v1
+    metadata:
+      name: azuredisk-csi-resizer-role
+      labels:
+        app.kubernetes.io/name: azuredisk-csi-driver
+        app.kubernetes.io/version: 1.23.0
+    subjects:
+      - kind: ServiceAccount
+        name: csi-azuredisk-controller-sa
+        namespace: kube-system
+    roleRef:
+      kind: ClusterRole
+      name: azuredisk-external-resizer-role
+      apiGroup: rbac.authorization.k8s.io
+  - kind: ClusterRole
+    apiVersion: rbac.authorization.k8s.io/v1
+    metadata:
+      name: csi-azuredisk-controller-secret-role
+      labels:
+        app.kubernetes.io/name: azuredisk-csi-driver
+        app.kubernetes.io/version: 1.23.0
+    rules:
+      - apiGroups: [""]
+        resources: ["secrets"]
+        verbs: ["get", "list"]
+  - kind: ClusterRoleBinding
+    apiVersion: rbac.authorization.k8s.io/v1
+    metadata:
+      name: csi-azuredisk-controller-secret-binding
+      labels:
+        app.kubernetes.io/name: azuredisk-csi-driver
+        app.kubernetes.io/version: 1.23.0
+    subjects:
+      - kind: ServiceAccount
+        name: csi-azuredisk-controller-sa
+        namespace: kube-system
+    roleRef:
+      kind: ClusterRole
+      name: csi-azuredisk-controller-secret-role
+      apiGroup: rbac.authorization.k8s.io
+  - kind: ClusterRole
+    apiVersion: rbac.authorization.k8s.io/v1
+    metadata:
+      name: csi-azuredisk-node-role
+      labels:
+        app.kubernetes.io/name: azuredisk-csi-driver
+        app.kubernetes.io/version: 1.23.0
+    rules:
+      - apiGroups: [""]
+        resources: ["secrets"]
+        verbs: ["get"]
+      - apiGroups: [""]
+        resources: ["nodes"]
+        verbs: ["get"]
+  - kind: ClusterRoleBinding
+    apiVersion: rbac.authorization.k8s.io/v1
+    metadata:
+      name: csi-azuredisk-node-secret-binding
+      labels:
+        app.kubernetes.io/name: azuredisk-csi-driver
+        app.kubernetes.io/version: 1.23.0
+    subjects:
+      - kind: ServiceAccount
+        name: csi-azuredisk-node-sa
+        namespace: kube-system
+    roleRef:
+      kind: ClusterRole
+      name: csi-azuredisk-node-role
+      apiGroup: rbac.authorization.k8s.io
+  - kind: DaemonSet
+    apiVersion: apps/v1
+    metadata:
+      name: csi-azuredisk-node
+      namespace: kube-system
+      labels:
+        app.kubernetes.io/name: azuredisk-csi-driver
+        app.kubernetes.io/version: 1.23.0
+    spec:
+      updateStrategy:
+        rollingUpdate:
+          maxUnavailable: 1
+        type: RollingUpdate
+      selector:
+        matchLabels:
+          app: csi-azuredisk-node
+      template:
+        metadata:
+          labels:
+            app.kubernetes.io/name: azuredisk-csi-driver
+            app.kubernetes.io/version: 1.23.0
+            app: csi-azuredisk-node
```

Comment:
following annotation is missing. refer
https://github.com/IBM/ibm-satellite-storage/blob/master/config-templates/azure/azuredisk-csi-driver/1.18.0/deployment.yaml#L311